hook.php

```diff
@@ -374,6 +376,24 @@ function plugin_formcreator_hook_update_ticket(CommonDBTM $item) {
          'items_id'     => $id
       ]
    ]);
+
+   // find the 1st requester
+   $requester = $DB->request([
+      'SELECT' => 'users_id',
+      'FROM' => Ticket_User::getTable(),
+      'WHERE' => [
+         'tickets_id' => $item->getID(),
+         'type' => CommonITILActor::REQUESTER,
+      ],
+      'ORDER' => ['id'],
+      'LIMIT' => '1',
+   ])->next();
```

Comment:
`->next()` do not return anything in GLPI 10 so `$requester` will be undefined.
Maybe use `current()` so the code work on both versions ?

Reply:
If I remember wel calling current() without previous call to next() will not work. That'sd why I plan to to a specific version of  the fix for branch 2.13 of Formcreator, specifically for this suggestion.

Reply:
You can even do something like `current($requesters)['users_id'] ?? 0;`

Reply:
@cedric-anne Is this the best way to have the same code working for both versions ?

Reply:
maybe `current(iterator_to_array($iterator))`